src/errors.ts

```diff
@@ -41,14 +41,6 @@ export class OutputValidationError extends IOSchemaError {
   constructor(public override readonly cause: z.ZodError) {
     super(getMessageFromError(cause), { cause });
   }
-
-  /**
-   * @deprecated use the cause property instead
-   * @todo remove in v21
-   * */
-  public get originalError() {
```

Comment:
add migration script

Reply:
to be sure we need to enforce type checking, but it complicates the whole thing significantly